omero/sysadmins/limitations.rst

```diff
@@ -81,6 +81,21 @@ described above.
 
 .. _minmax_limitation:
 
+Import of OME-NGFF
+^^^^^^^^^^^^^^^^^^
+
+The import of `OME-NGFF <https://ngff.openmicroscopy.org/latest/>`_ is currently limited to :doc:`command-line (CLI) importer </users/cli/import>` only.
+
+Naming of OME-NGFF images in OMERO
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default naming of the `OME-NGFF <https://ngff.openmicroscopy.org/latest/>`_ Plates/Images imported into OMERO is not intuitive at the moment. Use the ``-n NAME`` option of :doc:`command-line (CLI) importer </users/cli/import>` to achieve explicit naming.
```

Comment:
```suggestion
The default naming of the `OME-NGFF <https://ngff.openmicroscopy.org/latest/>`_ Plates/Images imported into OMERO is not intuitive at the moment. Use the ``-n NAME`` option of the :doc:`command-line (CLI) importer </users/cli/import>` to achieve explicit naming.
```

Reply:
Is it correct for plates or is this limitation only applicable to non-HCS data?

My understanding is the HCS Behavior is identical to the one currently and the `Plate.Name` metadata takes precedence.